client/src/containers/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -111,6 +114,7 @@ const SignUp: React.FC<Props> = ({ onSubmit }) => (
           label="Password*"
           outllined
           component={TextField}
+          id="password"
```

Comment:
Why do we need id property in these fields?

Reply:
1. This is material-ui thing. id property is getting transformed to 
```
<label for={valueInId} ></label>
<input id={valueInId} />
```
I think ` for` in labels is always good for semantic reasons
2. Thanks to that during tests we can in easy way query inputs by `getByLabeLText`